src/mono/sample/HelloWorld/Makefile

```diff
@@ -2,7 +2,7 @@ TOP=../../../../
 DOTNET:=$(TOP)./dotnet.sh
 DOTNET_Q_ARGS=--nologo -v:q -consoleloggerparameters:NoSummary
 
-MONO_CONFIG ?=Debug
```

Comment:
I don't see the benefit of keep changing this back and force between Debug and Release. As a developer, I would prefer this being set to Debug.

Reply:
I don't particularly care which it's set to, but it needs to be consistent.

Since both you and Zoltan seem inclined towards release, I can set that everywhere, just haven't had a chance to update the PR.

Reply:
I prefer Debug actually.

Reply:
I prefer Debug as well. You just simply revert this change.

Reply:
Er sorry, I mean to say inclined to Debug. I can't just revert this though—a bunch of places still have Release set, so I need to go through and sort those out in addition to reverting this. I'll update the PR tomorrow probably; I'm trying to avoid too much typing today due to hand pain.

Reply:
Sounds good. Hope your hand pain will go away soon.